webui/react/src/components/ModelRegistry.tsx

```diff
@@ -715,13 +748,13 @@ const ModelRegistry: React.FC<Props> = ({ workspace }: Props) => {
           pagination={getFullPaginationConfig(
             {
               limit: settings.tableLimit,
-              offset: settings.tableOffset,
+              offset: params.tableOffset,
             },
             total,
           )}
           rowClassName={defaultRowClassName({ clickable: false })}
           rowKey="name"
-          settings={settings}
+          settings={{ ...settings, tableOffset: params.tableOffset }}
```

Comment:
nit: maybe lift this into a memo -- creating a new object every render could cause problems down the line